javascript/javascript2/week3/index.html

```diff
@@ -0,0 +1,36 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>JS2 Week3</title>
+    <link rel="stylesheet" href="./game.css" />
+  </head>
+  <body>
+    <button id="delay-btn">Click</button>
+    <button id="location-btn">Log location</button>
+    <div id="your-location"></div>
+    <a id="find-on-map" target="_blank"></a>
+    <section class="input-and-buttons">
+      <input type="number" id="seconds" />
+      <button id="get-seconds" class="start">Start Game</button>
+      <button id="reset" class="reset">Reset Game</button>
+    </section>
+    <section class="press-button">
+      <span id="press-s" class="press-s"></span>
```

Comment:
```suggestion
      <span id="press-s" class="press-s">
          <p>S key</p>
      </span>
```

Reply:
It would be nice to have a paragraph for these spans, otherwise, it's impossible to know what this is about.